libaubio, bump version

Begasus opened the pull request.

alaviss commented on media-libs/libaubio/libaubio-0.4.8.recipe:
```diff
@@ -0,0 +1,88 @@
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	ffmpeg$secondaryArchSuffix
```
Does it require everything provided by ffmpeg? I think we should expand this into `lib:` entries.

Begasus commented:
Adjusted ffmpeg recipes, also removed the older recipes/patchset(s) (v0.4.1 also didn't build for gcc2), patchset has been upstreamed before current version
